### ISSUE TITLE
FedRAMP Rev4 Guidance is missing ac-8_fr_gdn.1

### ISSUE THREAD

**Issue by grosven** [usnistgov/oscal-content#73]
# Describe the bug

https://github.com/usnistgov/oscal-content/blob/master/fedramp.gov/json/FedRAMP_HIGH-baseline-resolved-profile_catalog.json

Missing  "Guidance: If performed as part of a Configuration Baseline check, then the % of items. requiring setting that are checked and that pass (or fail) check can be provided." 

# Who is the bug affecting?
Everyone relying on this file

# What is affected by this bug?
I can't rely on the data for generating the correct OSCAL Template
{Describe the impact the bug is having.}

# When does this occur?
n/a
{Describe the conditions under which the bug is occurring.}

# How do we replicate the issue?
Search "ac-8_fr_gdn.1"

If applicable, add screenshots to help explain your problem.}

# Expected behavior (i.e. solution)
Should be ac-8_fr_gdn.1 "Guidance: If performed as part of a Configuration Baseline check, then the % of items. requiring setting that are checked and that pass (or fail) check can be provided." 

# Other Comments

**Comment by ohsh6o** [comment]
@david-waltermire-nist, per our chat can you move this to [the GSA/fedramp-automation repo](https://github.com/GSA/fedramp-automation)? Thank you!

**Comment by ohsh6o** [comment]
I talked to @david-waltermire-nist and we cannot use the Github "transfer issue" function so I had to copy-paste manually. FYI, per #70 in the baseline has been removed from this repo. We can review [the updated baselines in the official GSA/fedramp-automation repo](https://github.com/GSA/fedramp-automation) and confirm whether this issue has been resolved or not. I and other FedRAMP staff will need to check. The baselines were not identical, now or then.

@david-waltermire-nist, you or @grosven can close this issue now.
